game.py

```diff
@@ -405,26 +478,109 @@ def _button_press_cb(self, win, event):
                     color=self._colors[0]))
             if self._opts[i].type == self._recall_list[self._answer]:
                 self._opts[i].set_label('☻')
-                self._correct += 1
+                self._correct += 2
+                self._correct_for_level += 1
             else:
                 self._opts[i].set_label('☹')
-                self._correct = 0
+                self._correct_for_level = 0
 
         if self._game in [0, 1, 3]:
             for i in range(self._level):
                 self._dots[i].set_layer(100)
         else:
+            for question_shape in self._question:
+                question_shape.hide()
             for dot in self._dots:
                 dot.set_shape(self._new_dot_surface(
                         image=self._recall_list[self._answer]))
                 dot.set_layer(100)
 
-        if self._correct == 0:
-            self._timeout_id = GLib.timeout_add(5000, self.new_game)
+        if self._correct > current:
+            self._timeout_id = GLib.timeout_add(1000, self.new_game)
         else:
-            self._timeout_id = GLib.timeout_add(3000, self.new_game)
+            self._timeout_id = GLib.timeout_add(2000, self._game_over)
         return True
-
+        
+    def _game_over(self):
+        for opt in self._opts:
+            opt.hide()
+            opt.type = -1
+        for dot in self._dots:
+            dot.hide()
+        for question_shape in self._question:
+            question_shape.hide()
+        self.save_highscore()
+        yoffset = int(self._space / 4.)
+        xoffset = int((self._width - 6 * self._dot_size - \
+                                   5 * self._space) / 2.)
+        y = 1
+        i=0
```

Comment:
Spaces are preferred around asignment operators
```suggestion
        i = 0
```